spec/cddl/many.cddl

```diff
@@ -118,15 +118,27 @@ error = {
 ; An identity is either a tagged byte string matching the identity format, or a textual
 ; representation of the identity (which can deserialize to the byte string matching the
 ; identity format).
-identity = tstr / #6.10000(
-    anonymous-identity /
-    public-key-identity /
-    subresource-identity
-)
+identity =
+    textual-non-anonymous-identity
+    / textual-anonymous-identity
+    / #6.10000(non-anonymous-identity)
+    / #6.10000(anonymous-identity)
+
+; A textual version of an identity that is anonymous.
+textual-anonymous-identity = "oaa" / "oaaaa"
+
+; A textual identity. This does not validate the actual content of the text (ie. the
+; checksum might be wrong).
+textual-identity = tstr .regexp "o[abcdefghijklmnopqrstuvwxyz234567]{2}[abcdefghijklmnopqrstuvwxyz234567]*"
+
+; An identity that isn't anonymous.
+non-anonymous-identity = public-key-identity / subresource-identity
 
 anonymous-identity = h'00'
 public-key-identity = bytes .cbor [ h'01', bytes .size 28 ]
 subresource-identity = bytes .cbor [ h'80'..h'FF', bytes .size 28, bytes .size 3 ]
 ; end::identity[]
 
-attribute = uint / [uint, * any]
+; An attribute can have one or many attribute arguments.
+attribute-arg = ()
```

Comment:
Should we make use of this in other documents?

Reply:
Yes, as part of a separate PR.